src/main/java/bowling_step3/domain/FrameLast.java

```diff
@@ -0,0 +1,54 @@
+package bowling_step3.domain;
+
+public class FrameLast extends FrameMutual implements Frame {
```

Comment:
```suggestion
public class FrameLast extends FrameMutual {
```

FrameMutual이 Frame 인터페이스를 구현하고 있다.
FrameMutual만 상속해도 되지 않을까?